modules/fundamental/src/organization/endpoints/test.rs

```diff
@@ -34,30 +38,23 @@ async fn all_organizations(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
         )
         .await?;
 
-    ctx.graph
-        .ingest_advisory(
-            "EMPORIUM-1",
-            ("source", "http://captpickles.com/"),
-            &Digests::digest("EMPORIUM-1"),
-            AdvisoryInformation {
-                id: "EMPORIUM-1".to_string(),
-                title: Some("EMPORIUM-1".to_string()),
-                version: None,
-                issuer: Some("Capt Pickles Boutique Emporium".to_string()),
-                published: Some(OffsetDateTime::now_utc()),
-                modified: None,
-                withdrawn: None,
-            },
-            &ctx.db,
-        )
-        .await?;
+    Ok(advisory)
+}
 
-    let uri = "/api/v2/organization?sort=name";
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn organization_tests(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
```

Comment:
I am against merging all tests into one. That makes it very hard to debug single failed tests.

Reply:
I agree with this part here :+1: